docs/guides/designing-the-delta-format.md

```diff
@@ -88,14 +88,14 @@ This canonicalization applies to both keys and values, `text` and `attributes`.
 - There is only one way to represent a newline which is with `\n`, not `\r` or `\r\n`
 - `text: "Hello  World"` unambiguously means there are precisely two spaces between "Hello" and "World"
 
-Some of these choices may be customized by the user, but the canonical contraint in Deltas dictate that the choice must be unique.
+Some of these choices may be customized by the user, but the canonical constraint in Deltas dictate that the choice must be unique.
 
-This unambiguous predictability makes Deltas easier to work with, both because you have fewer cases to handle, but also because there are no surprises in what a corresponding Delta well look like. Long term, this makes applications using Deltas easier to understand and maintain.
+This unambiguous predictability makes Deltas easier to work with, both because you have fewer cases to handle and because there are no surprises in what a corresponding Delta will look like. Long term, this makes applications using Deltas easier to understand and maintain.
 
 
 ## Line Formatting
 
-Line formats affect the contents of the entire line, so it present an interesting challenge for our compact and canonical constraint. A seemingly reasonable way to represent center aligned text is this:
+Line formats affect the contents of an entire line, so they present an interesting challenge for our compact and canonical constraints. This is a seemingly reasonable way to represent centered text:
```

Comment:
The reason I prefer "A seemingly reasonable way to represent center aligned text is this:" is the that pronoun "this" is at the end so there's no confusion what it is referring to. When it is at the beginning, the reader may wrongly associate it with a noun in the previous sentence.